interfaces/PA-App/src/app/personal-components/select-language/select-language.component.spec.ts

```diff
@@ -2,6 +2,7 @@ import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { TranslateModule } from '@ngx-translate/core';
 import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { PopoverController } from '@ionic/angular';
```

Comment:
This should not be necessary when using the "MoreInfoButtonComponent" via the "SharedModule"..